api-module-library/42matters/package.json

```diff
@@ -1,31 +1,28 @@
 {
-  "name": "@friggframework/api-module-42matters",
-  "version": "0.0.5",
-  "prettier": "@friggframework/prettier-config",
-  "description": "",
-  "main": "index.js",
-  "scripts": {
-    "lint:fix": "prettier --write --loglevel error . && eslint . --fix",
-    "test": "jest"
-  },
-  "author": "",
-  "license": "MIT",
-  "devDependencies": {
-    "@friggframework/assertions": "^1.0.21",
-    "@friggframework/eslint-config": "^1.0.20",
-    "@friggframework/logs": "^1.0.23",
-    "@friggframework/module-plugin": "^1.1.15",
-    "@friggframework/test-environment": "^1.1.20",
-    "dotenv": "^16.3.1",
-    "eslint": "^8.49.0",
-    "jest": "^29.7.0",
-    "prettier": "^3.0.3",
-    "sinon": "^16.0.0"
-  },
-  "publishConfig": {
-    "access": "public"
-  },
-  "dependencies": {
-    "md5": "^2.3.0"
-  }
-}
+    "name": "@friggframework/api-module-42matters",
+    "version": "0.0.5",
+    "prettier": "@friggframework/prettier-config",
+    "description": "",
+    "main": "index.js",
+    "scripts": {
+        "lint:fix": "prettier --write --loglevel error . && eslint . --fix",
+        "test": "jest"
+    },
+    "author": "",
+    "license": "MIT",
+    "devDependencies": {
+        "@friggframework/devtools": "^1.0.0",
+        "dotenv": "^16.3.1",
+        "eslint": "^8.49.0",
+        "jest": "^29.7.0",
+        "prettier": "^3.0.3",
+        "sinon": "^16.0.0"
+    },
+    "publishConfig": {
+        "access": "public"
+    },
+    "dependencies": {
+        "md5": "^2.3.0",
+        "@friggframework/core": "^1.0.0"
```

Comment:
Per Andrew's comment in Discord, should this be a local/workspace reference to let Lerna do the crawling? or are we intentionally version pinning?

Reply:
I'm happy to switch to that, I was going to try to find some examples before updating. 

What do you think? That would be like:

`"@friggframework/core": "packages:core"` 

or more literally:

`"@friggframework/core": "workspace:*"`

Reply:
@seanspeaks this one is the most important I think. But I won't wait on merge

Reply:
Worth a look probably module by module for our desired behavior as we iterate through stuff.

Tests will help. Then we can just run tests as we make changes and feel somewhat confident we didn't break everything 😂